### PR TITLE
[DO NOT MERGE] COMMERCE-11755 Testplan for COMMERCE-11162 Supplier Role to manage Price Lists and Promotions

### PR DESCRIPTION
**Pull request by pietrobocale**
https://liferay.atlassian.net/browse/COMMERCE-11755

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
Cherry-picked Brian Kim's commit to verify if the tests are passing (to remove when the commit is merged: https://github.com/brianchandotcom/liferay-portal/pull/137097)

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:stop

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 1]
```diff
@@ -655,4 +1767,288 @@ definition {
...
+			AssertElementPresent(
+				key_entryName = "Test Catalog Base Price List",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
+		}
+
+		task ("And the user can't see the Price Lists related to Minium") {
+			AssertElementNotPresent(
+				key_entryName = "Minium Base Price List",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:2001]:
```suggestion
		task ("Then the supplier can see only the Price List related to the new Catalog") {
			CommercePriceLists.openPriceListsAdmin();

			AssertElementPresent(
				key_entryName = "Test Catalog Base Price List",
				locator1 = "CommerceNavigation#ENTRY_NAME");
		}

			AssertElementNotPresent(
				key_entryName = "Minium Base Price List",
				locator1 = "CommerceNavigation#ENTRY_NAME");
		}
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 2]
```diff
@@ -655,4 +1767,288 @@ definition {
...
+	}
+
+	@description = "COMMERCE-11843. This a test to verify that the Supplier User can only see the Promotion that are related to him (COMMERCE-11757)"
+	@priority = 5
+	test CanSupplierUserViewOnlyRelatedCatalogPromotion {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:2006]: Please apply changes from the previous test to this one where necessary

**Comment by pietrobocale** [comment]
ci:test:sf

**Pull request closed without merging**